src/Core/Settings.cpp

```diff
@@ -1017,6 +1017,9 @@ When enabled, allows to use legacy toTime function, which converts a date with t
 Otherwise, uses a new toTime function, that converts different type of data into the Time type.
 The old legacy function is also unconditionally accessible as toTimeWithFixedDate.
 )", IMPORTANT) \
+    DECLARE(Bool, allow_experimental_time_time64_type, false, R"(
+Allows creation of [Time](../../sql-reference/data-types/time.md) and [Time64](../../sql-reference/data-types/time64.md) data types.
+)", EXPERIMENTAL, enable_time_time64_type) \
```

Comment:
It's not marked as `IMPORTANT` as per https://github.com/ClickHouse/ClickHouse/pull/75735#issuecomment-2815865059